packages/cli/src/commands/bundle/bundle.ts

```diff
@@ -4,16 +4,21 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @format
  */
-
+import {Config} from '@react-native-community/cli-types';
+import {CommandLineArgs} from './bundleCommandLineArgs';
 import buildBundle from './buildBundle';
 import bundleCommandLineArgs from './bundleCommandLineArgs';
```

Comment:
You can remove line 9 nad move the import here
```suggestion
import bundleCommandLineArgs, {CommandLineArgs} from './bundleCommandLineArgs';
```